openwisp-config/Makefile

```diff
@@ -94,4 +94,22 @@ define Package/openwisp-config/install
 	$(CP) ../VERSION $(1)/usr/lib/openwisp-config/
 endef
 
+# for backward compatibility
+define Package/openwisp-config/postinst
+#!/bin/sh
+if [ ! -L /usr/sbin/openwisp_config ]; then
+	ln -s /usr/sbin/openwisp-config /usr/sbin/openwisp_config
+fi
+
+if [ ! -L /etc/init.d/openwisp_config ]; then
+	ln -s /etc/init.d/openwisp-config /etc/init.d/openwisp_config
+fi
+endef
+
+define Package/openwisp-config/postrm
+#!/bin/sh
+rm -f /usr/sbin/openwisp_config
+rm -f /etc/init.d/openwisp-config
```

Comment:
This was a mistake:
```suggestion
rm -f /etc/init.d/openwisp_config
```

Reply:
Yes, I noticed that mistake, too. But apart from that, it looks good.